environment.sh

```diff
@@ -12,20 +12,24 @@ Help()
    echo "h     Print this Help."
    echo "v     Verbose mode."
    echo "t     Type of conda environment. Either 'simulation' (default) or 'artifact'."
+   echo "n     Make a new conda environment even if one currently exists."
 }
 
 # Define variables
 username=$(whoami)
 env_type="simulation"
+make_new="no"
 
 # Process input options
-while getopts ":ht:" option; do
+while getopts ":ht:n" option; do
```

Comment:
Ok I removed the v option now because it was not being handled and Im not sure the best way to handle different verbose modes in bash and I don't want people debugging this it should work so if there is a problem they can file a ticket instead of troubleshooting

Reply:
More importantly, the options for getopts needs a ':' after them if they are a flag that has an options and no ':' if they do not. So the h and n flags just trigger and action whereas the t flag needs to be artifact or simulation.